addons/hw_posbox_homepage/__manifest__.py

```diff
@@ -21,7 +21,7 @@
 """,
     'assets': {
         'web.assets_backend': [
-            'hw_posbox_homepage/static/img/*',
+            'hw_posbox_homepage/static/*/**',
```

Comment:
```suggestion
            'hw_posbox_homepage/static/**/*',
```
shouldn't it be like this?